<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>783</width>
    <height>737</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nodes</string>
    </property>
   </widget>
   <widget class="QListWidget" name="node_list">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>191</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="flowchart_list">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>191</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Flowcharts</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="node_information">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>791</width>
      <height>671</height>
     </rect>
    </property>
    <property name="title">
     <string>Data</string>
    </property>
    <widget class="QGroupBox" name="generic_information">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>271</width>
       <height>251</height>
      </rect>
     </property>
     <property name="title">
      <string>Generic Node Information</string>
     </property>
     <widget class="QLabel" name="type_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Type</string>
      </property>
     </widget>
     <widget class="QLabel" name="subtype_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Subtype</string>
      </property>
     </widget>
     <widget class="QLabel" name="subtype_value_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Subtype value</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="subtype_value_edit">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>90</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="id_edit">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>120</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="id_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="next_id_edit">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>150</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="next_id_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Next node ID</string>
      </property>
     </widget>
     <widget class="QLabel" name="table_index_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>String index</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="string_index_edit">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>180</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="subtype_edit">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>60</y>
        <width>191</width>
        <height>22</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="type_edit">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>30</y>
        <width>191</width>
        <height>22</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="add_next_node_button">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>251</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Add next node</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="string_information">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>20</y>
       <width>251</width>
       <height>391</height>
      </rect>
     </property>
     <property name="title">
      <string>String table information</string>
     </property>
     <widget class="QListWidget" name="strings_list">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>231</width>
        <height>221</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="string_index_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>index</string>
      </property>
     </widget>
     <widget class="QLabel" name="strings_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>181</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Strings</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="index_edit">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>270</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="edit_string_button">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>330</y>
        <width>231</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Edit string</string>
      </property>
     </widget>
     <widget class="QPushButton" name="string_reference_button">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>360</y>
        <width>231</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Find string reference</string>
      </property>
     </widget>
     <widget class="QPushButton" name="add_string_button">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>231</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Add string</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="message_information">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>420</y>
       <width>271</width>
       <height>221</height>
      </rect>
     </property>
     <property name="title">
      <string>Message Information</string>
     </property>
     <widget class="QLineEdit" name="label_edit">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="msbt_label_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>MSBT Label</string>
      </property>
     </widget>
     <widget class="QLabel" name="message_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Message</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="message_edit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>241</width>
        <height>131</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="specifc_information">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>261</width>
       <height>361</height>
      </rect>
     </property>
     <property name="title">
      <string>Node Specific Information</string>
     </property>
     <widget class="QLabel" name="param_1_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Parameter 1</string>
      </property>
     </widget>
     <widget class="QLabel" name="param_2_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Parameter 2</string>
      </property>
     </widget>
     <widget class="QLabel" name="param_3_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Parameter 3</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="param_3_edit">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>90</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="param_4_edit">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>120</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="param_4_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Parameter 4</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="param_2_edit">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>60</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="param_1_edit">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>30</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="branch_list_label">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>20</y>
        <width>181</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Branch cases</string>
      </property>
     </widget>
     <widget class="QListWidget" name="branch_list">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>241</width>
        <height>131</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="add_branch_button">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>320</y>
        <width>241</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Add branch</string>
      </property>
     </widget>
     <widget class="QLabel" name="branch_case_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Branch Cases</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="goto_root_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Go  to root</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_flowchart_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>191</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Add flowchart</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back_button">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>280</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Go back</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>783</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="file_menu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNewFile"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
   </widget>
   <addaction name="file_menu"/>
  </widget>
  <action name="actionNew_2">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionNewFile">
   <property name="text">
    <string>New File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionChange_to_little">
   <property name="text">
    <string>Little</string>
   </property>
  </action>
  <action name="actionBig">
   <property name="text">
    <string>Big</string>
   </property>
  </action>
  <action name="actionEncoding_2">
   <property name="text">
    <string>Encoding</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
